scripts/core/context.yml

```diff
@@ -197,6 +210,15 @@ params:
       name: hNativeContext
       desc: |
             [in] the native handle of the context.
+    - type: uint32_t
+      name: numDevices
+      desc: "[in] number of devices associated with the context"
+    - type: "const $x_device_handle_t *"
```

Comment:
If I had to guess, the python problem is because of the space before the asterisk. This should be `"const $x_device_handle_t *"` instead.

It might also be a good idea to fix the python script, but that's probably a bit more work.

Reply:
> This should be `"const $x_device_handle_t*"` instead.

Agree this looks like it should resolve the issue. I'll have a look at fixing the script separately.

Reply:
This extra space also seems to change the scripts understanding of this type to handle rather than the intended pointer to handle...

Reply:
thanks @kbenzie , @pbalcer ! yes, it was that :)